<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gventas</class>
 <widget class="QDialog" name="Gventas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Ventas</string>
  </property>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Lista de ventas</string>
   </property>
   <widget class="QTableView" name="lista2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>114</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout2">
     <item>
      <widget class="QPushButton" name="addbutton2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Adicionar Nueva Consignacion</string>
       </property>
       <property name="statusTip">
        <string>Adicionar Nueva Consignacion</string>
       </property>
       <property name="text">
        <string>Adicionar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDockWidget" name="informaciondventasadicionado">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>44</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="features">
        <set>QDockWidget::DockWidgetClosable</set>
       </property>
       <property name="windowTitle">
        <string>Informacion</string>
       </property>
       <widget class="QWidget" name="informacion2_2">
        <widget class="QLabel" name="informacionlabel2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Asignacion realizada 
 con exito</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>331</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="volverboton">
      <property name="toolTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="statusTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="dventasw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Nueva Venta</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>211</width>
        <height>106</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Producto:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="productocomboBox2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label2_2">
         <property name="text">
          <string>Cantidad:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="cantidad2"/>
       </item>
      </layout>
     </widget>
     <widget class="QDockWidget" name="informacionproductoadicionado">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>524287</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetClosable</set>
      </property>
      <property name="windowTitle">
       <string>Producto Adicionado</string>
      </property>
      <widget class="QWidget" name="informacion2_4"/>
     </widget>
     <widget class="QTableView" name="listadventas">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>381</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addproductoDventas">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Producto</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label2_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addventas">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
      </property>
     </widget>
     <widget class="QSpinBox" name="id2spinbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>47</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="codven2spinbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>47</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="costo2spinbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="total2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/dialog-cancel.png</normaloff>:/images/dialog-cancel.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>181</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="ventaid_6">
        <property name="text">
         <string>Condven:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dventasid">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ventaid_3">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="dventasfecha">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ventaid_7">
        <property name="text">
         <string>Total en bs:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="dventastotal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="dventasdetalle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6"/>
  </widget>
 </widget>
 <resources>
  <include location="pandasoft.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>volverboton</sender>
   <signal>clicked()</signal>
   <receiver>Gventas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
